docs/dev-tools/mcp-server.md

```
@@ -183,104 +338,1142 @@ uvx --version
+| ------------------ | ----------- | -------- |
+| `project_id`       | UUID string | **Yes**  |
+| `name`             | string      | **Yes**  |
+| `description_html` | string      | No       |
+
+---
+
+## Common workflows
+
+### Look up a work item by ID
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)